code/modules/shuttle/computers/dropship_computer.dm

```diff
@@ -142,16 +142,22 @@
 	if(shuttle.mode == SHUTTLE_CRASHED)
 		to_chat(user, SPAN_NOTICE("\The [src] is not responsive"))
 		return
-
+	var/remaining_time = timeleft(door_control_cooldown) / 10
+	to_chat(user, SPAN_WARNING("The shuttle is not responding, try again in [remaining_time] seconds."))
```

Comment:
At the moment this message sort of suggests we can't do anything about it. If it was done in character more it probably wouldn't have this effect; but I'm not sure what would be automatically done to allow it to fix itself (that we can speed up).

Also the estimates should probably qualified with `about` or `around` even if they actually are totally accurate timer amounts because it would be weird to know precisely how many seconds you need to clear off goop.

Reply:
I think it's better now. Also I made a for cycle instead of notorious while(true)